userservice/src/main/java/com/infodation/userservice/repositories/UserRepository.java

```diff
@@ -17,9 +17,10 @@ public interface UserRepository extends JpaRepository<User, Long> {
             "WHERE :name = '' OR u.firstName LIKE %:name% OR u.lastName LIKE %:name%")
     Page<User> findByName(@Param("name") String name, Pageable pageable);
 
+    @Query("SELECT u.userId FROM users u")
+    List<String> findAllUserIds();
```

Comment:
You can return the Set<String> directly here, so then you no need to cast again `List` to `Set` at https://github.com/thaibinh156/user-service/pull/11/files#diff-d6ab78c7eb4c57c14c360014ef30971bc491c852e8a7caa8557c30793411363cR33-R34